RF_Receiver/RF_Receiver.ino

```diff
@@ -457,6 +460,12 @@ void send_mc(const uint8_t startpos,const uint8_t endpos, const int16_t clock)
 
 	}
 
+  // generate none manchester conform signal to mark the end
+  stoptime = micros() + (isHigh(PIN_SEND) ? 5 : 4);
```

Comment:
Ist in Anlehnung an culfw. Dort wird ein zusätzliches clock gewartet, wenn der Pegel nach dem letzten Bit high isst.